test/create-new-project.e2e.test.ts

```diff
@@ -15,8 +15,8 @@ import { retryCommand } from "./utils/retry-command";
 import { forceExitAllCommands, runCommand } from "./utils/run-command";
 
 const PROJECT_NAME = "kit-demo";
-const TEMPLATE_NAME = "kit-asset-tokenization";
-const SUBGRAPH_NAMES = ["kits", "kit-asset-tokenization"];
+const TEMPLATE_NAME = "asset-tokenization-kit";
```

Comment:
should be asset-tokenization.

Should be a valid id of a template
<img width="1282" alt="Screenshot 2025-03-07 at 14 49 43" src="https://github.com/user-attachments/assets/804371b0-b2b4-43c7-a13d-9136360a41d2" />